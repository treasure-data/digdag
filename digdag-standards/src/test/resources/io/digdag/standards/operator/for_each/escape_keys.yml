_command:
  '!?': [1, 2, 3]

_do:
  echo>: hello world
