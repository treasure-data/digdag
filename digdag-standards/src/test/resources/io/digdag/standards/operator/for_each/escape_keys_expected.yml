"+for-0=%21%3F=0=1":
  echo>: hello world
  _export: {'!?': 1}
"+for-0=%21%3F=1=2":
  echo>: hello world
  _export: {'!?': 2}
"+for-0=%21%3F=2=3":
  echo>: hello world
  _export: {'!?': 3}
